utils/constants.js

```diff
@@ -20,12 +20,12 @@ export const daysOfWeek = [
 export const modesOfTransport = [
   "Bicycle",
   "Bus",
+  "Carpool",
+  "Motorbike",
+  "Taxi Ride",
   "Car",
-  "Motorbike/scooter",
-  "Taxi/ride share",
+  "Walk", 
```

Comment:
I prefer we use `By foot` instead of 'Walk' (if we need to shorten this)
at least, 'By foot' won't exclude ppl who do jog/run as part of their commute.
```suggestion
  "By foot", 
```

Reply:
This is on discussion but no words from figma team.